content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -233,6 +233,11 @@ To use the `systemd` cgroup driver in `/etc/containerd/config.toml` with `runc`,
 The `systemd` cgroup driver is recommended if you use [cgroup v2](/docs/concepts/architecture/cgroups).
 
 {{< note >}}
+The elipses above indicate that config.toml should already have these sections. If you
```

Comment:
did you mean "ellipsis" instead of "elipses"?
i would avoid the word either way, and use something more common.

but....instead of amending this note which is about plugins, how about adding a new note on top of the example TOML saying in the lines of "run command FOO  if your TOML file does not exist yet"?

Reply:
elipses is strictly correct, but it's an obscure word.